### PR TITLE
Update npm scripts, remove deprecated sendfile

### PR DESCRIPTION
**Pull request by acabunoc**
* no need to watch in production
* add clean
* sendfile -> sendFile (deprecated in express)

### CODE REVIEW

**Comment by acabunoc** [comment]
Small change -- @alicoding can you r? THANK YOU THANK YOU

File: app.js [hunk 1]
```diff
@@ -195,7 +195,7 @@ app.get('/papers/:doi/badges/:orcid/badges/:badge', function(request, response){
 
 
 app.get('*', function(request, response) {
-  response.sendfile(__dirname + '/public/index.html');
+  response.sendFile(__dirname + '/public/index.html');
```
Comment (alicoding) [inline app.js]: oops!

**Comment by alicoding** [comment]
r+

File: app.js (hunk 1 above)
Comment (acabunoc) [inline app.js]: haha, express told me sendfile was deprecated

**Pull request merged**